.ci/packer_cache.sh

```diff
@@ -18,3 +19,8 @@ if [ -x "$(command -v docker)" ]; then
   (retry 2 docker pull "${image}") || echo "Error pulling ${image} Docker image, we continue"
   done
 fi
+
+# Download Go module dependencies.
+go mod download
```

Comment:
If this is not required then `|| true` for each command otherwise the cache might fail and affect the cache image generations for some other projects.

Reply:
TIL, thanks. Good thing you pointed this out, since there was an error with one of the `go mod download` lines.